legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-ansi/src/test/java/org/finos/legend/engine/persistence/components/ingestmode/bitemporal/BitemporalDeltaWithBatchIdTest.java

```diff
@@ -1295,6 +1460,13 @@ void testMilestoningSourceSpeciesFromWithDataSplitFilterDuplicates()
         Assertions.assertEquals(getExpectedCleanupSql("\"mydb\".\"temp\"", "temp"), operations.get(0).ingestSql().get(5));
         Assertions.assertEquals(getExpectedCleanupSql("\"mydb\".\"stagingWithoutDuplicates\"", "stage"), operations.get(0).ingestSql().get(6));
 
+        String incomingRecordCount = "SELECT COUNT(*) as incomingRecordCount FROM \"mydb\".\"staging\" as stage WHERE (stage.\"data_split\" >= 2) AND (stage.\"data_split\" <= 5)";
```

Comment:
We can have a placeholder for lower and upper bound and then replace it with values.
That way we can reuse the same set of SQL.